src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -7,7 +7,228 @@ service: Payload CMS
 i18nReady: false
 ---
 
-[Payload](https://payloadcms.com/) is a headless CMS and application framework.
+[PayloadCMS](https://payloadcms.com/) is a headless open-source content management system that can be used to provide content for your Astro project.
+
+## Integrating with Astro
+
+## Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+2. **A MongoDB database** - PayloadCMS will ask you for a MongoDB connection string when creating a new project. You can set one up locally or use [MongoDBAtlas](https://www.mongodb.com/) to host a database on the web for free.
+3. **A PayloadCMS REST API** - Create a [PayloadCMS](https://payloadcms.com/) project and connect it to your MongoDB database during the setup.
+
+### Installing PayloadCMS
```

Comment:
```suggestion
```
So I've been thinking about this, and since we already said a Payload project was a prerequisite, maybe the note just directly follows that. And everything necessary to connect Astro is what should be the next heading here?